integration_tests/e2e/frontendComponentsFallback.cy.ts

```diff
@@ -26,7 +26,7 @@ context('Frontend Components Fallback', () => {
 
   describe('Header', () => {
     it('should display the correct details for the signed in user', () => {
-      cy.task('stubSignIn')
+      cy.task('stubSignIn', { name: 'Bobby Brown' })
```

Comment:
we're now using the sign in token to get the name, rather than manage users call